flottform/forms/src/create-flottform-input.ts

```diff
@@ -410,34 +439,30 @@ export function createFlottformInput({
 			if (!didReceiveSomething) {
 				changeState('receiving-data');
 			}
-			if (!hasMetaInformation) {
-				const fileMeta = JSON.parse(e.data) as {
-					lastModified?: number;
-					name?: string;
-					size: number;
-					type?: string;
-				};
-				size = fileMeta.size;
-				fileName = fileMeta.name ?? fileName;
-				fileType = fileMeta.type ?? fileType;
-				hasMetaInformation = true;
+
+			if (!fileMeta) {
+				fileMeta = JSON.parse(e.data) as FileMetaInfos;
```

Comment:
We should use zod or similar here to ensure `fileMeta` is a valid `FileMetaInfos` type. Needs a new issue to track this. The new issue should include a test for it as we need to check what happens if it isn't valid (this function will throw - what about its state then?)